src/runtimes/node/index.js

```diff
@@ -53,6 +55,7 @@ const zipFunction = async function ({
   }
 
   return zipEsbuild({
+    archiveFormat,
     config,
     destFolder,
     destPath,
```

Comment:
This line can be removed.

Reply:
Absolutely. The path is now computed in the individual zip functions. Fixed in https://github.com/netlify/zip-it-and-ship-it/pull/410/commits/b29fd723a3db7b82cbeb128c1ea70714a441042c.